### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by CaffeyChen** [cassiePython/NeRF-Art#7]
Hi，
will you release the training code soon? or could I request a copy of the implement of the criteria ？thanks！

**Comment by cassiePython** [comment]
You can send me an email. I will share you with a copy of the training code and loss functions.

**Comment by CaffeyChen** [comment]
Thanks a lot!
